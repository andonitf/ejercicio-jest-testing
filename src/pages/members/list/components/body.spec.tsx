import * as React from 'react';
import { shallow } from 'enzyme';
import { Body } from './body';

describe('pages/members/list/components/body specs', () => {
    it('should render the component as expected when passing required properties', () => {
        // Arrange
        const props = {
            members: [
                { id: 1, name: 'test name 1', avatarUrl: 'http://some/avatar/url/1'},
            ],
        };

        // Act
        const component = shallow(<Body members={props.members} />);

        // Assert
        expect(component).toMatchSnapshot();
    });
});
