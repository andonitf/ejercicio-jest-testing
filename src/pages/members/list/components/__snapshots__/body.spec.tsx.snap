// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/members/list/components/body specs should render the component as expected when passing required properties 1`] = `
<tbody>
  <Row
    key="1"
    member={
      Object {
        "avatarUrl": "http://some/avatar/url/1",
        "id": 1,
        "name": "test name 1",
      }
    }
  />
</tbody>
`;
